styles.css

```diff
@@ -0,0 +1,733 @@
+color: white;
+position: relative;
+
+}
+.x-div-alert3
+{
+position: absolute;
+right: 8px;
+top:-2px;
+color: white;
+font-size: 18px;
+font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+}
+.icon-button
+{
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 80px;
+    color: #333333;
+    background: white;
+   border-radius: 50%;
+   border: none;
+    outline: none;
+    
+}
+.icon-button__badge
+{
+    position: absolute;
+    top: -3px;
+    right:22px;
+    width: 15px;
+    height: 15px;
+    background: red;
+    color: white;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border-radius: 50%;
+}
+.sub-heading
+{
+    font-style: italic;
+    color: var(--primary-color);
+}
+.primary-button
+{
+    padding: 0.5rem;
+    color: white;
+    background-color: var(--primary-color);
+    border-radius: 0.5rem;
+    border: none;
+    outline: none;
+    font-size: 15px;
+}
+.icon-button-buttons
+{
+    padding:0.5rem;
+    color: white;
+    background-color: rgb(40, 185, 3);
+    border: none;
+    outline: none;
+    font-size: 15px;
+   
+}
+.float-action-button
+{
+    padding: 0.5rem;
+    color: white;
+    background-color: var(--dark-pink);
+    border-radius: 45%;
+    border: none;
+    outline: none;
+    font-size: 15px;
+}
+.link-button{
+    padding: 0.5rem;
+    color: rgb(3, 163, 184);
+    border-radius: 1rem;
+    background-color: white;
+    border-color: rgb(3, 163, 184);
+    font-size: 15px;
+}
+.secondary-button
+{
+    padding: 0.5rem;
+    color: var(--primary-color);
+    background-color: white;
+    border-radius: 0.5rem;
+     border-color: var(--primary-color);
+     outline:none;
+     font-size: 15px;
+}
+.success-badge-button
+{
+    padding: 0.5rem;
+    color: white;
+    background-color: green;
+    border-radius: 0.5rem;
+    border: none;
+    outline: none;
+    margin-right: 1rem;
+}
+.danger-badge-button
+{
+    padding: 0.5rem;
+    color: white;
+    background-color: red;
+    border-radius: 0.5rem;
+    border: none;
+    outline: none;
+    margin-right: 1rem;
+}
+.warning-batch-button
+{
+    padding: 0.5rem;
+    color: white;
+    background-color:rgb(207, 207, 12);
+    border-radius: 0.5rem;
+    border: none;
+    outline: none;
+    margin-right: 1rem;
+}
+
+.sub-div1-frontPage
+{
+    padding: 5rem;
+
+}
+.h2-frontPage
+{
+    font-family: 
+    'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+    font-size:40px;
+    color: var(--primary-color);
+    font-weight: bold;
+    text-align: center;
+}
+.small-frontPage
+{
+    font-family: 
+    'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+    font-size:23px;
+    padding: 0.5rem 0rem 0rem 4rem;
+    color: var(--dark-pink);
+}
+.button-frontPage
+{
+    padding: 1rem;
+    text-align: center;
+
+}
```

Comment:
It's better to add `cursor: pointer` to all your buttons. It makes the user feel of interactivity.